huawei/huaweifusionsolardiscovery.cpp

```diff
@@ -46,15 +46,11 @@ void HuaweiFusionSolarDiscovery::startDiscovery()
     qCInfo(dcHuawei()) << "Discovery: Start searching for Huawei FusionSolar SmartDongle in the network...";
     NetworkDeviceDiscoveryReply *discoveryReply = m_networkDeviceDiscovery->discover();
 
-    // Check any already discovered infos..
-    foreach (const NetworkDeviceInfo &networkDeviceInfo, discoveryReply->networkDeviceInfos()) {
-        checkNetworkDevice(networkDeviceInfo);
-    }
-
```

Comment:
given this is not needed any more, you can also remove the entire m_verifiedNetworkDevices member in all the plugins...

Reply:
also this whole block in finished() can be removed
```
        // Send a report request to nework device info not sent already...
        foreach (const NetworkDeviceInfo &networkDeviceInfo, m_networkDeviceInfos) {
            if (!m_verifiedNetworkDeviceInfos.contains(networkDeviceInfo)) {
                checkNetworkDevice(networkDeviceInfo);
            }
        }
```
In fact, I think the m_networkDeviceInfos member variable is not needed any more either for the majority of the plugins (depends a bit how the post-processing looks like)

Reply:
what about this?

Reply:
should all be fixed, see the diff underneeth